package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "ion-study-template",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "rollup -c",
+    "start": "npm run build && web-ext run",
+    "lint": "eslint . && web-ext lint"
```

Comment:
It would be nice to have one that does `web-ext build` too (maybe "package"?) Could split this out to a separate issue if you want, we should align whatever we do with things in the core repo like mozilla-ion/ion-core-addon/issues/135

Reply:
I think instead of adding “package” script, it might make more sense to modify “build” to use `roll up -c && web-ext build`, since I think both need to be run together (otherwise you might accidentally build the add-on with unrolluped code, or run rollup but then not rebuild the add-on (although running `npm start` might address the latter concern).

Reply:
I added the `npm run package` command: `npm run build` still allows us to _quickly_ bundle stuff to test in a browser, which is slightly different from package (which generates a zip without the unneeded files).